mz_crypt.c

```diff
@@ -37,8 +37,6 @@ uint32_t mz_crypt_crc32_update(uint32_t value, const uint8_t *buf, int32_t size)
    /* Define z_crc_t in zlib 1.2.5 and less or if using zlib-ng */
 #  if (ZLIB_VERNUM < 0x1270)
     typedef unsigned long z_crc_t;
-#  else
-    typedef uint32_t z_crc_t;
 #  endif
```

Comment:
z_crc_t was added in:
- zlib 1.2.7 (https://github.com/madler/zlib/commit/6c9bd474aa08312ef2e2e9655a80e18db24a1680) as `unsigned long`
- zlib-ng 2.1.0 (https://github.com/zlib-ng/zlib-ng/commit/ff9162c1a0da8d60ae664e5930b256de99ad83b3) as `uint32_t`

They were added with conflicting types, so we should not redefine them.